client/src/modules/invoices/registry/templates/action.cell.html

```diff
@@ -34,6 +34,11 @@
         <span class="fa fa-money"></span> <span translate>REPORT.VIEW_PAYMENTS</span>
       </a>
     </li>
+    <li>
+      <a data-method="viewInvoice" ui-sref="journal({ filters : [{ key: 'record_uuid', value : row.entity.uuid}]})" href>
```

Comment:
At the moment, all of these will show a giant uuid at the top ... which isn't very pretty.  @lomamech was working on fixing this in #2139.

You can prevent introducing new ones in your PR by using `displayValue` in the filters.  This will pass a value to be displayed.  The `value` will be used to filter.